### PR TITLE
Remove usage from Readme and add developer advice

### PR DESCRIPTION
**Pull request by sirkrypt0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,20 +4,9 @@
 This is an implementation of a text editor in Squeak, which is capable of writing structured text. This means that text can be divided into logical units. Paragraphs are used to structure the text into sections on a broad level and so-called "text structures" are used to assign certain meanings to particular parts of the text.
 
 # Using the editor
-## Paragraphs
-Pressing *Enter* inserts a Parapraph, pressing *Shift + Enter* insert a normal line break. The paragraph is visible by an increased line height.
-Structures can be applied to portions of the text by selecting the particular text and selecting the structures from the list of available structures on the left-hand side.
+You can launch SquidWord by going to the `Apps` tab in Squeaks menu bar and clicking on `SquidWord`.
```
Comment (frcroth) [inline README.md]:
```suggestion
Once installed, you can launch SquidWord by going to the `Apps` tab in Squeaks menu bar and clicking on `SquidWord`.
```

File: README.md [hunk 2]
```diff
@@ -4,20 +4,9 @@
...
-
-## Saving and Loading
-When the text is saved via the *Save* menu item or by pressing *Ctrl + s*, the changes regarding the text and structures are written into a **RichTextDocument** object. This object can be inspected by clicking the *Document* button in the editor. It is possible to open the document in another editor by sending the message *openInEditor* to it.
-
-The *Save as...* menu item lets the user choose a name for the document. The document is then saved in the image (in it's current state, so make sure to save temporary changes to the document prior to that). It can be loaded in any other editor via the *Load* menu item.
-
-## Import and Export
-The document can be exported into a file and imported from a file. At the current moment, text structures can not be imported or exported.
+For information on using the editor and it's functionalities, refer to the integrated `Help Browser` that can be opened using the `Help` button in the upper right hand corner of SquidWord.
```
Comment (frcroth) [inline README.md]:
```suggestion
For information on using the editor and its functionalities, refer to the integrated `Help Browser` that can be opened using the `Help` button in the upper right hand corner of SquidWord.
```

File: packages/RichTextEditing-Core.package/SquidWordHelp.class/class/savingAndLoading.st [hunk 3]
```diff
@@ -0,0 +1,20 @@
+pages
+savingAndLoading
+	"This method was automatically generated. Edit it using:"
+	"SquidWordHelp edit: #savingAndLoading"
+	^(HelpTopic
+		title: 'Saving and Loading'
+		contents: 
+'Save
```
Comment (frcroth) [inline packages/RichTextEditing-Core.package/SquidWordHelp.class/class/savingAndLoading.st]: Should go into some detail to explain the seemingly weird behavior of save.

File: packages/RichTextEditing-Core.package/SquidWordHelp.class/class/keyboardShortcuts.st [hunk 4]
```diff
@@ -7,13 +7,16 @@ keyboardShortcuts
 		contents: 
 'Structuring
 Ctrl-Shift-[1,2,...,0]	toggle 1st, 2nd, ... or 10th structure from the top of the structure list
+(NOTE: This works only when using an american keyboard layout)
```
Comment (frcroth) [inline packages/RichTextEditing-Core.package/SquidWordHelp.class/class/keyboardShortcuts.st]: Should propably fix that by creating another preference and different shortcut string

File: packages/RichTextEditing-Core.package/SquidWordHelp.class/class/exportAndImport.st [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+PlainText
+This export strategy simply exports your written text without any structural elements like applied structures. The result is by default a .txt file.
+
+HTML
+This export strategy allows you to export your written and structured text to a HTML format. To avoid any problems make sure that you use the predefined *html* structure template available in the structure editor. If for whatever reason you choose to not use the predefined template, SquidWord will try to translate your structures but there is no guarantee this will work.
+The result is by default a .html file that can be viewed e.g. in your browser.
+
+Markdown
+This export strategy is currently just like the HTML export strategy with the sole difference that it exports your file to a .md file. The reason is that such HTML file is valid Markdown and that Markdown provides much less functionality using it''s integrated formatting options than rich text. There might be an export to ''real'' Markdown in the future.
```
Comment (frcroth) [inline packages/RichTextEditing-Core.package/SquidWordHelp.class/class/exportAndImport.st]: You can add text about Serialized object here as well

**Review by frcroth — CHANGES_REQUESTED** [review]

File: packages/RichTextEditing-Core.package/SquidWordHelp.class/class/keyboardShortcuts.st (hunk 4 above)
Comment (sirkrypt0) [reply packages/RichTextEditing-Core.package/SquidWordHelp.class/class/keyboardShortcuts.st]: good point

Comment (sirkrypt0) [reply packages/RichTextEditing-Core.package/SquidWordHelp.class/class/keyboardShortcuts.st]: See #178

**Review by oleschl — APPROVED** [review]

**Pull request merged**